packages/wrangler/src/api/integrations/bindings/index.ts

```diff
@@ -93,6 +98,7 @@ export async function getBindingsProxy<Bindings = Record<string, unknown>>(
 			...vars,
 			...bindings,
 		},
+		cf: Object.freeze(await mf.getCf()),
```

Comment:
Arguably we should move the `freeze` to miniflare 🤔 

what do you think @mrbbot?

Reply:
As discussed we've kept the `freeze` and actually implemented it recursively as that matches more closely workerd 🙂